src/xdocs/config_filters.xml

```diff
@@ -737,6 +737,13 @@ public class UserService {
           Note, that support for other Checks will be available after resolving
           <a href="https://github.com/checkstyle/checkstyle/issues/4830">issue 4830</a>.
         </p>
+        <p>
+          You can use the command line helper tool to generate xpath
+          suppressions based on your configuration file and input class file.
+          See
+          <a href="http://checkstyle.sourceforge.net/cmdline.html">here</a>
```

Comment:
This doesn't need to be full path. `cmdline.html` should be fine.

Reply:
done